index.html

```diff
@@ -46,717 +46,789 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
-									             Offline Reinforcement Learning.</a> Kumar et al. (2020)</li>
-									<li> <a href="https://arxiv.org/abs/2104.08212"> COMBO: Conservative Offline 
-									             Model-Based Policy Optimization.</a> Yu et al. (2021)</li>
-								</ul>
-							
-							</td>
-						</tr>
-						<tr class="active">
-							<td><b>Week 5</b><br />
-								Wed, May 3
-							</td>
-							<td><i><span class="label label-lecture text-base">Lecture</span></i> 
-								<a href="slides/cs224r_offline_rl_2.pdf">Offline RL Part 2</a>
-							</td>
-							<td>
-								<span class="label label-due text-base">Due</span></i><b> Homework 2</b><br>
-								Homework 3 out
-								[<a href="material/cs224r_hw3_2022.pdf">PDF</a>, <a href="material/hw3.zip">code, <a href="material/hw3_template.zip">template</a></a>]
-							</td>
-							<td>
-								<ul>
-									<li> <a href="https://arxiv.org/abs/2110.06169"> Offline Reinforcement Learning 
-									              with Implicit Q-Learning.</a> Kostrikov et al. (2021)</li>
-								</ul>
-							</td>
-						</tr>
-						<tr>
-							<td><b>Week 6</b><br />
-								Mon, May 8
-							</td>
-							<td><i><span class="label label-lecture text-base">Lecture</span></i>
-								<a href="slides/cs224r-mtrl.pdf">Multi-Task RL and Goal-Conditioned RL</a>
-						    </td>
-							<td>
-							</td>
-							<td>
-							
-								<ul>
-									<li> <a href="https://arxiv.org/abs/1707.01495"> Hindsight Experience Replay.</a> Andrychowicz et al. (2017)</li>
-									<li> <a href="https://arxiv.org/abs/2104.08212"> MT-Opt: Continuous Multi-Task Robotic Reinforcement Learning at Scale</a> Kalashnikov et al. (2021)</li>
-									<li> <a href="https://arxiv.org/abs/2001.06782"> Gradient Surgery for Multi-Task Learning.</a> Yu et al. (2020)</li>
-								</ul>
-							
-							</td>
-						</tr>
-						<tr>
-							<td><b>Week 6</b><br />
-								Wed, May 10
-							</td>
-							<td><i><span class="label label-guest text-base">Guest Lecture</span></i> 
-    							<a href="slides/cs224r-RLLeggedRobots.pdf">Transfer Learning in RL (Jie Tan)</a>
-    						</td>
-							<td>
-							<!--
+                        </td>
+                        <td>
+
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/1312.5602"> Playing
+                                        Atari with Deep Reinforcement Learning.</a> Mnih et al. (2013)</li>
+                            </ul>
+
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 3</b><br />
+                            Fri, April 18
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a" href="slides/cs224r_q_learning_section.pdf">Extra section on Q-learning</a>
```

Comment:
Should have a note that mentions time and location for this (maybe before the papers)